### DISCUSSION TITLE
PyTorch Live

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#138] [General]
**新聞內容:**
Meta發布了一套稱為PyTorch Live的工具，讓開發者能夠在**行動裝置上開發人工智慧應用程式**，並且可與廣泛的PyTorch社群交流，不只簡化行動裝置的生產環境研究，也支援透過聯合學習來保護使用者隱私。目前仍在Beta版本，僅支援MacOS，很快就會新增Windows和Linux的支援。
PyTorch Live其包含了PyTorch Live命令列介面，以及能夠以PyTorch Mobile執行裝置上推理的React Native套件，還有一些React Native範例模板，供開發者可以快速部署到行動裝置上。
使用PyTorch Live，建置出人工智慧應用程式非常簡單，經過命令列工具引導就能快速地設定環境，並且開始機器學習行動裝置專案開發，針對現有的應用程式，也能夠簡單地整合裝置上模型，目前可支援圖片分類、問答、手寫數字辨識，或是其他用戶自定義的模型。
**新聞心得:**
有上這個官網上看他們的介紹影片，除了有辨識功能之外，好像還能在自己已經建立好的程式上使用！喔買尬，Meta才改名沒幾天就公布新進展了！
**新聞來源:**
https://www.ithome.com.tw/news/148244
![live](https://user-images.githubusercontent.com/90883526/145214369-53c540f8-ed08-4a31-9b68-2cf128a309e5.jpg)

**Comment by pocper1** [comment]
Meta 快把人工智慧藉由手機終端裝置，廣泛的應用了，再也不用利用電腦開發人工智慧的應用程式了

**Comment by wh20502** [comment]
感覺要建置AI程式還是喜歡在電腦上面使用，一方面螢幕較大，一方面操作也熟悉，可能新一代的國高中生接受度會很高。
